backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java

```diff
@@ -143,6 +144,8 @@ protected void loop() {
 				if (window.shouldClose()) {
 					closedWindows.add(window);
 				}
+				if (window.getConfig().foregroundFPS > targetFramerate)
```

Comment:
considering 0 (maxFPS) as the highest value, if 2 windows are created with 60 fps, user won't be able to change it to 0.

i'm not sure taking the maximum is the best strategy considering FPS are "shared" by all windows, maybe we could instead use a convention where first window is the master window and take value from this one?